### PR TITLE
Add react slider

### PR DESCRIPTION
**Pull request by wdelmas**

### CODE REVIEW

File: src/clients/mobiles/app.tsx [hunk 1]
```diff
@@ -10,6 +11,7 @@ export interface AppProps {
 
 const App = (props: AppProps) => {
     return <Remote {...props} />
+    // return <Tabs {...props} />
```
Comment (bcldvd) [inline src/clients/mobiles/app.tsx]: 🤔

File: src/clients/mobiles/tabs/index.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+export interface TabsProps {
+    socketService: SocketService
+}
+
+class TabsContainer extends React.Component<TabsProps & ReduxStore, {}>  {
+
+    public newTab() {
+        this.props.socketService.sendTabsActionsMessageFromClient({
+            actionType: OPEN_NEW_TAB,
+            action: 'https://www.dashlane.com'
```
Comment (bcldvd) [inline src/clients/mobiles/tabs/index.tsx]: 🤨

**Comment by bcldvd** [comment]
🤯🤩
So many cool stuff :))

File: src/clients/mobiles/app.tsx (hunk 1 above)
Comment (wdelmas) [reply src/clients/mobiles/app.tsx]: Just a test component working to send Tap action messages.
I wanted to keep it for testing in the future

File: src/clients/mobiles/tabs/index.tsx (hunk 2 above)
Comment (wdelmas) [reply src/clients/mobiles/tabs/index.tsx]: Yeah not really generic for now ^^^
But the point was to have something working

**Pull request merged**